Contents.swift

```diff
@@ -1,11 +1,30 @@
 import Cocoa
 
+// create an array of various shapes
 let shapes: Array<Shape> = [
     Rectangle(withHeight: 1, withWidth: 2),
-    Rectangle(withHeight: 2, withWidth: 3),
-    Rectangle(withHeight: 3, withWidth: 4)
+    Triangle(withSideA: 5, withSideB: 5, withBase: 6, withHeight: 4),
+    Square(withSide: 4),
```

Comment:
Is there a reason you've changed from three rectangles to rectangle, triangle, and square?